__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const app = require("../app")
+const request = require("supertest")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data/index")
+const db = require("../db/connection")
+
+beforeEach(() => {
+    return seed(data)
+})
+  
+afterAll(() => {
+    return db.end()
+})
+
+describe('/api/topics', () => {
+    test("200: Should return a status 200", () => {
```

Comment:
This 200 status is being asserted as returned in your following test too so no need to have it as a separate test.